node/node_syncing.go

```diff
@@ -40,6 +40,11 @@ func (node *Node) getNeighborPeers(neighbor *sync.Map) []p2p.Peer {
 	return tmp
 }
 
+// GetSync gets sync-ed to blockchain without joining consensus
+func (node *Node) GetSync() {
```

Comment:
GetSync is not good naming. DoSyncing is better.

Reply:
Usually Get* is a getter func, should return the stuff to get.

Reply:
`DoSyncing` already exists. `GetSync` just wraps around `DoSyncing`.

Reply:
Then it should be DoSyncingWithoutConsensus. GetSync is confusing.

Reply:
Changed.